src/coll/mod.rs

```diff
@@ -632,16 +632,8 @@ impl Namespace {
             coll: String::new(),
         }
     }
-}
-
-impl fmt::Display for Namespace {
-    fn fmt(&self, fmt: &mut fmt::Formatter) -> fmt::Result {
-        write!(fmt, "{}.{}", self.db, self.coll)
-    }
-}
 
-impl<'de> Deserialize<'de> for Namespace {
```

Comment:
I worry that this change could potentially break user's code that were relying on it. What about just augmenting this implementation to support both formats? @saghm